src/main/java/frc/robot/Constants.java

```diff
@@ -42,6 +42,37 @@ public static enum Mode {
         REPLAY
     }
 
+    public class RobotConstants {
+        public static String serial;
+        public static boolean isComp;
+        public static boolean isAlpha;
+    
+        static {
+            if (Robot.isReal()) {
+                // Roborio id recognition
+                serial = System.getenv("serialnum");
+            } else {
+                serial = "3467";
+            }
+            RobotConstants.isComp = serial.startsWith(RobotConstants.compSerial);
+            RobotConstants.isAlpha = serial.startsWith(RobotConstants.alphaSerial);
+        }
+        
+        // TODO: Fill in with real serial number prefixes
+        public static final String compSerial = "0000";
+        public static final String alphaSerial = "3467";
+    }
+
+    public static RobotType robotType = RobotConstants.isComp ? RobotType.COMP
```

Comment:
Reformat this to be a bit more readable, putting : at the end of the line instead of start